public/index.html

```diff
@@ -3,27 +3,46 @@
 	<head>
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<link rel="icon" href="favicon.ico">
+		<link rel="icon" href="assets/image/favicon.ico">
 
 		<title>OpenFisca - Turn law into software</title>
 
-		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
 	</head>
 	<body>
-		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
+		<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
 
 		<h1>Turn law into software</h1>
 
 		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
-
-		<h3>Calculate citizens rights</h3>
-		<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
-
-		<h3>Predict company taxes</h3>
-		<p>Model any kind of entity and evaluate their eligibility to fiscal obligations and benefits.</p>
-
-		<h3>Assess the impact reforms could have</h3>
-		<p>Compare law as it is now and as it could be.</p>
+		
+		<div class="band">
+			<h3>Calculate citizens' rights</h3>
+			<p>
+				From an individual situation, calculate any variable of a national tax and benefit system.
+				<video class="screenshot" src="assets/video/mes-aides.mp4" poster="assets/image/mes-aides.png" controls></video>
+				<a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> example uses the OpenFisca Web API to calculate OpenFisca-France benefits.
```

Comment:
“example“ and “calculate OpenFisca-France” sounds confusing here. What about something like “Mes Aides uses OpenFisca and its web API to allow French citizens to calculate the benefits they are entitled to”?